Modelica/Magnetic/FluxTubes/Material/SoftMagnetic/NickelIron/MuMetall.mo

```diff
@@ -1,12 +1,14 @@
 within Modelica.Magnetic.FluxTubes.Material.SoftMagnetic.NickelIron;
-record MuMetall "MUMETALL (77% NiFe)"
+record MuMetall "Mu-metal (77% NiFe)"
```

Comment:
In the next major release the record name shall be changed from `MuMetall` to `MuMetal` by a conversion script.

Refs #3995